.github/workflows/e2e-lint.yaml

```diff
@@ -0,0 +1,23 @@
+name: 'e2e Pull Request Lint'
+on:
+  pull_request:
+    branches: [main]
+    paths:
+      - "src/e2e/**"
+  workflow_dispatch:
+
+jobs: 
+  App-lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think this is needed of a checkout step

Reply:
Also, __usually__ the token is injected automatically in this case.

Reply:
@NoorChasib You still need the checkout step here, you just don't have to give it the GITHUB TOKEN, which in most cases is automatically passed to the action

Reply:
I know, its updated already!